src/api/web3.js

```diff
@@ -93,6 +93,65 @@ const getNetworkId = async web3 => {
   }
 }
 
+const connectToInjectedWeb3 = () => {
+  let web3
+  if (window.ethereum) {
+    web3 = new Web3(window.ethereum)
+  } else if (window.web3 && window.web3.currentProvider) {
+    web3 = new Web3(window.web3.currentProvider)
+    networkState.readOnly = false
+  } else {
+    throw new Error('No web3 injected from browser')
+  }
+  return web3
+}
+
+const connectToLocalNode = lazyAsync(async () => {
```

Comment:
Is there a reason this has lazyAsync wrapped around it but the others functions do not?

Reply:
I wrapped this in `lazyAsync` as all the existing async functions were wrapped in that way. (except `getNetworkId` as `lazyAsync` can't handle as it takes a parameter)

`connectToInjectedWeb3`, `connectToInjectedWeb3` and `connectToCloudNode`don't require the use of await so I just left them as sync functions for simplicity.